conftest.py

```diff
@@ -0,0 +1,10 @@
+import sys
+
+
+def pytest_ignore_collect(path):
+    if sys.version_info[0] > 2:
+        if str(path).endswith("__py2.py"):
```

Comment:
Given there are no such files yet, why handle py2? I'd say remove it until we need. YAGNI.